geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/S2IndexTest.scala

```diff
@@ -0,0 +1,233 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.accumulo.index
+
+import java.util.Date
+
+import org.geotools.data.{Query, Transaction}
+import org.geotools.filter.text.ecql.ECQL
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.TestWithDataStore
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.index.conf.QueryHints.{BIN_BATCH_SIZE, BIN_LABEL, BIN_SORT, BIN_TRACK, SAMPLE_BY, SAMPLING}
+import org.locationtech.geomesa.utils.bin.BinaryOutputEncoder
+import org.locationtech.geomesa.utils.bin.BinaryOutputEncoder.BIN_ATTRIBUTE_INDEX
+import org.locationtech.geomesa.utils.collection.SelfClosingIterator
+import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.Filter
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class S2IndexTest extends TestWithDataStore {
```

Comment:
Hmm...  As a ticket for later, I wonder if we ought to refactor our index tests...  This is probably a copy of lots of other tests, no?

Reply:
i copied it from z2, but yeah i think it's just those 2 right now